doc/source/create-k8s-cluster.rst

```diff
@@ -233,7 +266,8 @@ Setting up Kubernetes on Microsoft Azure Container Service (AKS)
 
    1. You have to `not use RBAC <security.html#use-role-based-access-control-rbac>`_, since AKS does not support it
       yet.
-   2. You should skip step 2 (granting RBAC rights) :ref:`when setting up helm <helm-rbac>`.
+   2. You cannot grant RBAC rights on Azure, so double-check that you are running
```

Comment:
"You cannot grant RBAC rights on Azure" also seems wrong, since RBAC is entirely no working there, so it seems redundant. Also, it's only true for Azure AKS.